### DISCUSSION TITLE
UI not loading properly

### DISCUSSION THREAD

**Discussion by adricom231** [nextcloud/all-in-one#5600] [Questions] [answered]
### Steps to reproduce
1.Docker container of AIO
2.V29.8.0 and V30.0

3.

### Actual behavior 
When i access the site ( i use cloudflare tunnel) the only thing that loads is the background and the logo. 
It happends  in the loging screen and also to logged users
I tried migrating to beta version of aio but it didnt help.
If you want to see for yourself the url is https://cloud.bams.ovh/

#### Host OS 
Debian 12  inside of Proxmox
#### Output of `sudo docker info`

Client: Docker Engine - Community
 Version:    27.3.1
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc.)
    Version:  v0.17.1
    Path:     /usr/libexec/docker/cli-plugins/docker-buildx
  compose: Docker Compose (Docker Inc.)
    Version:  v2.29.7
    Path:     /usr/libexec/docker/cli-plugins/docker-compose

Server:
 Containers: 18
  Running: 13
  Paused: 0
  Stopped: 5
 Images: 36
 Server Version: 27.3.1
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Using metacopy: false
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: systemd
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 7f7fdf5fed64eb6a7caf99b3e12efcf9d60e311c
 runc version: v1.1.14-0-g2c9f560
 init version: de40ad0
 Security Options:
  apparmor
  seccomp
   Profile: builtin
  cgroupns
 Kernel Version: 6.1.0-26-amd64
 Operating System: Debian GNU/Linux 12 (bookworm)
 OSType: linux
 Architecture: x86_64
 CPUs: 2
 Total Memory: 5.786GiB
 Name: debil
 ID: 32900ac2-07ff-4695-a623-bff517129b50
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

#### Docker run command or docker-compose file that you used
sudo docker run \
--init \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 8080:8080 \
--env APACHE_PORT=11000 \
--env APACHE_IP_BINDING=0.0.0.0 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
nextcloud/all-in-one:latest

**Comment by szaimen** [comment, marked as answer]
Hi, please read carefully through https://github.com/nextcloud/all-in-one?tab=readme-ov-file#notes-on-cloudflare-proxytunnel

**Reply by adricom231** [reply]
Hi, ive checked everything , ive read carefully throught the notes , i tried just experimenting with the settings and nothing worked so i decided to reinstall from scratch because it stopped responding to any requests ( just noticed when looking at the screenshots from portainer that the redis container dissapered)
The weird thing is that it worked before and after updating to V29.8.0 it slowly started to be more corrupted and then it just stopped working at all

**Reply by szaimen** [reply]
Did you see this?

![Screenshot_20241115_190049_Brave](https://github.com/user-attachments/assets/11b190a1-0b2f-4275-bfe4-0f1b603c082a)

**Reply by adricom231** [reply]
It fixed the login prompt not loading but now the css can't load properly
![image](https://github.com/user-attachments/assets/00afe9e9-fca4-47cc-84a6-24b8f4d3892a)
![image](https://github.com/user-attachments/assets/c8a03a61-4c34-4872-a310-e1f64fb7be32)

**Reply by adricom231** [reply]
Ive found a glich
When the new user is created in V30.02 the ui doesnt load for them just like in the reply above
And to make it display normaly you just have to go to apperance and accesibility and then select any color theme and then the UI works and displays normaly.
